apps/bajour/src/components/frage-des-tages/fdt-poll-block.tsx

```diff
@@ -140,15 +140,19 @@ export const FdtPollBlock = ({poll}: {poll?: ApiV1.PollBlock['poll']}) => {
 
   const autoVote = useCallback(async () => {
```

Comment:
Wondering, how we could move this auto-vote functionality to poll block provider to make url voting available to all media by default.
We could add this to the Publishers Documentation. By now the provider would miss the pollId. Any idea?

Reply:
I would not add this to a provider as it's a side-effect and would be very hidden and thus hard to debug or requires knowledge.

I would rather have a central place where we would handle all of these URL shenanigans. Best to wait until we have the app router so we can do these things on the server side not the client. Now that we lost the ability to do proper middleware as we are not using AWS/Vercel.